com.adsbynimbus.nimbus/Runtime/Plugins/iOS/NimbusManager.swift

```diff
@@ -38,6 +38,10 @@ import NimbusSDK
 import NimbusSDK
 import UnityAds
 #endif
+#if NIMBUS_ENABLE_MOBILEFUSE
```

Comment:
It might be a good idea to consolidate all the NimbusSDK imports into a single one with the different SDK demands in an or condition. You might even want to define a macro for that condition like `NIMBUS_ENABLE_SDK_DEMAND = NIMBUS_ENABLE_MOBILEFUSE || NIMBUS_ENABLE_ADMOB` etc.